simplq/src/components/common/CreateJoinForm/CreateJoinForm.jsx

```diff
@@ -61,18 +56,14 @@ const CreateJoinForm = (props) => {
         />
       </div>
       <div className={styles['button-group']}>
-        {createInProgress ? (
-          <LoadingIndicator />
-        ) : (
-          <>
-            <div>
-              <StandardButton onClick={handleCreateClick}>Create Queue</StandardButton>
-            </div>
-            <div>
-              <StandardButton onClick={handleJoinClick}>Join Queue</StandardButton>
-            </div>
-          </>
-        )}
+        <Loading isLoading={actionStatus === 'pending'}>
```

Comment:
`LoadingStatus` component solves this common problem. One less import,  one less selector.

```jsx
<LoadingStatus dependsOn="createQueue">Content</LoadingStatus>
```

Reply:
https://github.com/SimplQ/simplQ-frontend/pull/553/files#r579805848